corehq/apps/reports/standard/project_health.py

```diff
@@ -162,7 +163,8 @@ def get_unhealthy_users(self):
         """
         if self._previous_summary:
             unhealthy_users = filter(
-                lambda stub: stub.is_active and not stub.is_performing,
+                lambda stub: stub.is_active and not stub.is_performing and not
+                CommCareUser.get(stub.user_id).is_deleted(),
```

Comment:
is there a possibility that `CommCareUser` will not find the user? (gets deleted or something?)